Dashboard skeleton

mattchx opened the pull request:
<img width="1318" alt="Screen Shot 2021-01-12 at 21 18 00" src="https://user-images.githubusercontent.com/55170649/104397877-a83d4e00-551b-11eb-85c9-9e06258c8948.png">

Note: This is a draft and is not fully complete.

**Components included:**
NavBar(Links + Profile info)
Title
SelectTabs
  EventTypesTab(contains profile info, CreateNewType Btn, and Event Cards)
  ScheduledEventTab( Empty and could be disabled)
Getting Started Button

**Issues needed to be handled:**
- add 3 icons: plus, clock, and setting/edit
- convert card component to accept props for: color bar, length of meeting, and number of participants.
- used a mix of Box(flexbox) and grid. May switch all to grid.
- added secondary color(Orange) to theme.js, I have yet to implement in all components
- must increase responsive design for different screen sizes & add navbar for mobile
- need to replace avatar with photo, adjust white space, font-sizes, and colors

_I appreciate any and all feedback!_

mattchx committed:
Basic working skeletion

Make up of:
NavBar(Links + Profile info)
Title
SelectTabs
- EventTypesTab(contains profile info, CreateNewType Btn, and Event Cards)
- Scheduled Event Tab( Empty and could be disabled)
Getting Started Button

mattchx committed:
Removed some useless code

dproc96 commented on client/src/components/Header/NavBar.js:
```diff
@@ -0,0 +1,73 @@
+  <AppBar className={classes.appBar} position='static'>
+        <Toolbar>
+          <Typography variant='h6' className={classes.title}>
+            <span className={classes.orangeText}>calend</span>App
```
In place of this you could use the logo image that was provided, would be easier to match the mocks that way

mattchx replied:
Thanks Dan! I'm on it.

dproc96 reviewed (commented):
So far this looks good, pending the changes you mentioned already. There are two things I noticed if you want to even closer match the mocks: the text in the navbar could be a little smaller and more spaced and the user's name in event types could be bolded.

These are nitpicks though, great work so far!

mattchx committed:
Make small changes

- added logo and user images
- fixed some spacing and font sizes
- streamlined some css code

mattchx committed:
Finishing touches

Make a number of minor tweaks.
- Fixed white space around components.
- Changed buttons size and shape
- Shortened size of tabs highlighter.
* Cards are still in prototype and will be completed later

mattchx commented:
<img width="1305" alt="Screen Shot 2021-01-13 at 21 41 44" src="https://user-images.githubusercontent.com/55170649/104537627-531b3e00-55e8-11eb-91a2-16346f8826cf.png">

I'm happy with how the elements all came together. Let me know if there are any issues!
Edit: I was not able to soften the NavBar shadow nor did I align the User image under event types.

dproc96 commented on client/src/components/Header/NavBar.js:
```diff
@@ -0,0 +1,91 @@
+    <AppBar className={classes.root} position='static'>
+      <Toolbar>
+        <span className={classes.title}>
+          <img width='110rem' src={Logo} />
```
Be sure to add an alt attribute for these images

dproc96 reviewed (approved):
Be sure to add the alt attributes to your images before merging, but once you've done that this looks good to go!

mattchx committed:
Added alt attributes to img and Avatar elements

mattchx committed:
Correct selected Tab color

mattchx committed:
Merge branch 'dev' into dashboard-skeleton
